.github/workflows/deploy-after-release.yml

```diff
@@ -0,0 +1,119 @@
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+
+      - name: Build function app
+        shell: bash
+        working-directory: ./src/Functions/Altinn.Auth.AuditLog.Functions
+        run: dotnet build --configuration Release --output ./output
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: function-app
+          path: ./src/Functions/Altinn.Auth.AuditLog.Functions/output
+
+  deploy-tt02:
+    name: Deploy to TT02
+    runs-on: ubuntu-latest
+    environment: TT02
+    needs:
+      - build
+
+    permissions:
+      id-token: write
+      contents: read
+      packages: read
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Download built function-app
+        uses: actions/download-artifact@v4
+        with:
+          name: function-app
+          path: ./artifacts/function-app
+
+      - name: Azure Login
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+      - uses: ./.github/actions/deploy
+        name: Deploy
+        with:
+          image-tag: ${{ github.sha }}
+          resource-group: ${{ vars.CONTAINER_APP_RESOURCE_GROUP_NAME }}
+          container-app: ${{ vars.CONTAINER_APP_NAME }}
+          function-app: ${{ vars.AZURE_FUNCTIONAPP_NAME }}
+          function-app-path: ./artifacts/function-app
+      
+      - name: Add Version tag to Docker Image
+        uses: shrink/actions-docker-registry-tag@v4
+        with:
+          registry: ${{ env.REGISTRY }}
+          repository: '${{ env.REPOSITORY }}'
+          target: ${{ github.sha }}
+          tags: ${{ github.event.release.tag_name }}
+
+  deploy-prod:
+    environment: PROD
+    runs-on: ubuntu-latest
+    needs: deploy-tt02
+    permissions:
+        id-token: write
+        contents: read
+        packages: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Download built function-app
+        uses: actions/download-artifact@v4
+        with:
+          name: function-app
+          path: ./artifacts/function-app
+
+      - name: Azure Login
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+      - uses: ./.github/actions/deploy
+        name: Deploy
+        with:
+          image-tag: ${{ github.sha }}
+          resource-group: ${{ vars.CONTAINER_APP_RESOURCE_GROUP_NAME }}
+          container-app: ${{ vars.CONTAINER_APP_NAME }}
+          function-app: ${{ vars.AZURE_FUNCTIONAPP_NAME }}
+          function-app-path: ./artifacts/function-app
```

Comment:
```suggestion
          function-app-path: ./artifacts/function-app

```

Reply:
dont understand what is the change. I see that both lines are same